usr/lib/common/new_host.c

```diff
@@ -3380,20 +3380,17 @@ CK_RV SC_DigestEncryptUpdate(STDLL_TokData_t *tokdata,
                              CK_ULONG ulPartLen, CK_BYTE_PTR pEncryptedPart,
                              CK_ULONG_PTR pulEncryptedPartLen)
 {
-    UNUSED(sSession);
-    UNUSED(pPart);
-    UNUSED(ulPartLen);
-    UNUSED(pEncryptedPart);
-    UNUSED(pulEncryptedPartLen);
+    CK_RV rc;
 
```

Comment:
Hmm, should we check if pulEncryptedPartLen == NULL? Or would SC_EncryptUpdate handle this?

Reply:
I would leave this to SC_EncryptUpdate, which does check for NULL.